### PR TITLE
NPM release + prettifying

### PR DESCRIPTION
**Pull request by alongalky**

### CODE REVIEW

File: commands/eureka-tasks.js [hunk 1]
```diff
@@ -13,16 +13,20 @@ const printError = require('../errors/print-error')
...
     const table = new Table({
-      head: ['Task Name', 'Machine', 'Status', 'Command', 'Tier', 'Start Time', 'Cost', 'Duration']
+      head: ['Machine', 'Status', 'Command', 'Tier', 'Cost', 'Duration'],
+      style: {
+        head: ['green'],
+        compact: true
+      }
     })
 
     for (const task of response) {
```
Comment (karimodm) [inline commands/eureka-tasks.js]: I think we should sort order the table by initialization timestamp. You could just do a `Array.prototype.sort`

File: config/config.js [hunk 2]
```diff
@@ -2,14 +2,26 @@ const os = require('os')
...
+    throw new Error(`Missing Eureka configuration file in ${filename}. Please contact Eureka team for on-boarding instructions`)
+  }
 
-module.exports = data
+  const data = yaml.safeLoad(fs.readFileSync(filename, 'utf8'))
+  module.exports = data
+} catch (err) {
+  const prefix = chalk.red('ERROR') + ` cannot read config file from ${chalk.blue(filename)}`
+  if (err.mark) {
+    // Markdown library error
```
Comment (karimodm) [inline config/config.js]: Do you mean YAML syntax error?

File: package.json [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
   },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/alongalky/eureka-cli-node"
+  },
   "keywords": [],
-  "author": "",
-  "license": "ISC",
+  "author": "Eureka",
+  "license": "MIT",
```
Comment (karimodm) [inline package.json:14]: Free software YAY!

**Review by karimodm — CHANGES_REQUESTED** [review]

File: config/config.js (hunk 2 above)
Comment (alongalky) [reply config/config.js]: Yes, but they call it markdown in the YAML library (hence the err.mark thing). I'll make the comment clearer

File: package.json (hunk 3 above)
Comment (alongalky) [reply package.json:14]: Church of emacs

**Pull request merged**